config/prod/phil-cough-data.yaml

```diff
@@ -0,0 +1,42 @@
+# Provision a general S3 bucket or a Synapse External Bucket
+# http://docs.synapse.org/articles/custom_storage_location.html
+template_path: remote/s3-bucket.yaml
+stack_name: cough-data
```

Comment:
this and filename should match

Reply:
Ugh, I wish there was a way to make this apart of the pre-commit check 🤦